samples/deployment/test-network/installFPC.sh

```diff
@@ -23,8 +23,12 @@ PEERS=("peer0.org1.example.com" "peer0.org2.example.com")
 ERCC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 ECC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 
-CC_ID="echo"
```

Comment:
I think we should keep the var name `CC_ID` instead of changing it to `TEST_CC_ID`

Reply:
That would also remove the need to rename all the variables in this changeset.

Reply:
This will be covered in #605